### DISCUSSION TITLE
Welcome to ac-training-lab Discussions!

### DISCUSSION THREAD

**Discussion by sgbaird** [AccelerationConsortium/ac-training-lab#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sgbaird** [comment]
I'm Sterling, the Director of Training and Programs at the Acceleration Consortium. I am in charge of setting up the AC Training Lab. Rachel Keunun, our lab manager, has been instrumental in procuring the items for the training lab.

**Comment by jesuscapistran** [comment]
I'm Jesús Capistrán, a postdoctoral researcher in Mexico. I am working on developing photovoltaic materials and solar cells by solution deposition. I believe in the Acceleration Consortium's mission to accelerate the design and discovery of new materials for Renewable Energy and Drug Discovery. 

To enhance my scientific workflow, I will use Python, LLMs, and 3D printing (I own a Prusa-MK4). 

About me: https://jesuscapistran.com/about/
